src/util/string.h

```diff
@@ -486,7 +481,7 @@ void str_replace(std::string &str, char from, char to);
  *
  * @see string_allowed_blacklist()
  */
-inline bool string_allowed(const std::string &str, const std::string &allowed_chars)
+inline bool string_allowed(std::string_view str, const std::string &allowed_chars)
```

Comment:
`string_view::find_first_not_of` does not need `std::string`.
```suggestion
inline bool string_allowed(std::string_view str, std::string_view allowed_chars)
```